xbmc/guilib/GUIBaseContainer.cpp

```diff
@@ -1306,10 +1306,21 @@ std::string CGUIBaseContainer::GetLabel(int info) const
     break;
   case CONTAINER_ACTUAL_ITEMS:
   case CONTAINER_NUM_ITEMS:
+  case CONTAINER_NONFOLDER_ITEMS:
     {
       unsigned int numItems = GetNumItems();
       if (info == CONTAINER_NUM_ITEMS && numItems && m_items[0]->IsFileItem() && (std::static_pointer_cast<CFileItem>(m_items[0]))->IsParentFolder())
         label = StringUtils::Format("%u", numItems-1);
+      else if (info == CONTAINER_NONFOLDER_ITEMS)
```

Comment:
This looks a bit odd to me. Why did you add CONTAINER_NONFOLDER_ITEMS to this case and do a second and third check? Each call will check for that case, then it'll do the unnecessary check (first if condition) before it'll check CONTAINER_NONFOLDER_ITEMS again. Why not add a separate case?